extensions/wikia/ArticleVideoMVP/styles/premium-mvp.scss

```diff
@@ -0,0 +1,135 @@
+#premium-mvp-article-header {
+	display: block;
+}
+
+#premium-mvp-video {
+	background-position: center center;
+	background-size: cover;
+	cursor: pointer;
+	margin-bottom: 11px;
+	padding-top: 56%;
+	position: relative;
+	width: 100%;
+
+	&:before {
+		background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), #000000);
+		bottom: 0;
+		content: '';
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 50%;
+	}
+
+	.video-title {
+		bottom: 41px;
+		color: #fff;
+		font-size: 24px;
+		left: 38px;
+		position: absolute;
+	}
+
+	.video-time {
+		bottom: 73px;
+		color: #656e78;
+		font-size: 14px;
+		font-weight: bold;
+		left: 38px;
+		position: absolute;
+		text-transform: uppercase;
+	}
+
+	.video-play-button {
+		background-image: url("/extensions/wikia/ArticleVideoMVP/images/play-button-solid.svg"); /* $wgCdnStylePath */
+		background-repeat: no-repeat;
+		background-size: contain;
+		bottom: 15px;
+		height: 120px;
+		position: absolute;
+		right: 15px;
+		width: 120px;
+	}
+
+	.video-thumbnail {
+		.close {
+			content: url("/extensions/wikia/ArticleVideoMVP/images/close.svg"); /* $wgCdnStylePath */
+			display: none;
+			padding: 7px;
+			position: absolute;
+			top: 0;
+			left: 0;
+			z-index: 1;
+		}
+	}
+
+	&.collapsed {
+		.video-container {
+			bottom: 40px !important;
```

Comment:
Important always suggest there is a problem. Do we really need to override values with `!important`? Why?